src/common/components/registerAccountModal/registerAccountModal.component.js

```diff
@@ -165,6 +165,8 @@ class RegisterAccountModalController {
         } else {
           // Proceed to Step 3
           this.stateChanged('contact-info')
+          // Ensure modal is large enough to display contact info form
+          this.setModalSize('md')
```

Comment:
Is this necessary? I think `this.stateChanged` is supposed to handle this. If it doesn't, we should probably update the logic there.

Reply:
I removed this logic with my new changes